src/diffusers/pipelines/unidiffuser/pipeline_unidiffuser.py

```diff
@@ -34,6 +35,11 @@
 
 # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_img2img.preprocess
 def preprocess(image):
+    warnings.warn(
+        "The preprocess method is deprecated and will be removed in a future version. Please"
+        " use VaeImageProcessor.preprocess instead",
+        FutureWarning,
+    )
```

Comment:
So, our plan is to refactor this in a future PR, yes?

Reply:
No this function should be fully deprecated and removed in the future